README.md

```diff
@@ -368,7 +368,7 @@ You can test core functionality by running:
 You can also test interoperability with following modules:
 
 - [ngx_coolkit](http://github.com/FRiCKLE/ngx_coolkit),
-- [ngx_echo](github.com/agentzh/echo-nginx-module),
+- [ngx_echo](http://github.com/agentzh/echo-nginx-module),
```

Comment:
This module has been moved under `openresty/` from `agentzh/`. Will you fix that too please?